src/sections/footer.html

```diff
@@ -1,4 +1,37 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="lg:hidden text-darkgreen text-[0.938rem] md:text-[1rem] font-medium">
+    <div class="md:flex justify-between md:mb-[1.313rem]"> 
+        <div id="footer__address" class="mb-[1.313rem]">
+            Agentur Baumeister <br>
+            Böckhstraße 13 <br>
+            10967 Berlin
+        </div>
+        <div id="footer__contact" class="mb-[7rem] md:mb-0 md:text-right">
```

Comment:
Use default tailwind values here. We don't need arbitrary values.
```suggestion
        </div>
        <div id="footer__contact" class="mb-28 md:mb-0 md:text-right">
```